Lib/test/test_stat.py

```diff
@@ -223,6 +224,25 @@ class TestFilemodeCStat(TestFilemode, unittest.TestCase):
     format_funcs = TestFilemode.format_funcs | {'S_ISDOOR', 'S_ISPORT',
                                                 'S_ISWHT'}
 
+    @cpython_only
+    def test_mode_t_converter(self):
+        from _testcapi import USHRT_MAX
```

Comment:
This is needed only on Windows. It is worth to split the test on two parts: the general one and the Windows-specific one.